### DISCUSSION TITLE
v0.4.0

### DISCUSSION THREAD

**Discussion by kpcyrd** [kpcyrd/libredefender#17] [General]
- Concurrency is now configurable
- The missing notification system has been added
- `libredefender scan .` is now working correctly even if `skip_hidden` is enabled

## Thanks

We'd like to thank @SantiagoTorres and @repi for their support on [github sponsors](https://github.com/sponsors/kpcyrd).

<hr /><em>This discussion was created from the release <a href='https://github.com/kpcyrd/libredefender/releases/tag/v0.4.0'>v0.4.0</a>.</em>

**Comment by mruiz42** [comment]
How is this better than just using clamav daemon? I like the name but it seems to be a wrapper over clamav. What features or improvements over clamav scheduled scans? You should make this clear in the README. Not trying to give off a bad tone, genuinely curious as I have it installed on my machine but I'm not sure it's purpose versus just running clamav. 

Also, I really do like the name of this, hopefully more features or AV engines can be utilized instead of just clamav, but I don't think there are many alternatives.

**Reply by kpcyrd** [reply]
hi, thanks for asking! It's essentially a clamav frontend similar to clamtk that I wrote because I had trouble getting clamtk to work.

**Comment by akenaton1123** [comment]
Hi, 
How can I update the virus database? Tks

**Reply by kpcyrd** [reply]
`sudo systemctl enable --now clamav-freshclam.service`
